domain/src/main/java/life/qbic/usermanagement/User.java

```diff
@@ -145,6 +146,15 @@ public String getEmail() {
     return this.email;
   }
 
+  /**
+   * Checks if a given password is correct for a user
+   * @param password Password that is being validated
+   * @return true, if the given password is correct for the user
+   */
+  public Boolean checkPassword(String password) {
```

Comment:
I would prefer not to pass clear text passwords around in the application. Instead, pass the encrypted password as argument.

You can get the encrypted password hash from the User object you bind the input data from the form :)

Reply:
But that's in the user class.. I need to create